requirements.txt

```diff
@@ -1,30 +1,2 @@
-# for debian build-deps, see the apt-get commands in
-# tools/vagrant-provision.sh
-
-psycopg2==2.4.5  # requires libpq to build
-Django==1.7.1
-IPy==0.81
-twisted>=12,<15
```

Comment:
There is one slight conflict here again, since Sigmund upped the minimum requirement for twisted in another PR